mozjs/makefile.cargo

```diff
@@ -54,13 +54,21 @@ ifneq ($(HOST),$(TARGET))
 			$(NULL)
 	endif
 
+	ifeq (armv7-unknown-linux-gnueabihf,$(TARGET))
+		# Reset TARGET variable because armv7 target name used by Rust is not
+		# the same as the target name needed for the CXX toolchain.
+		TARGET = arm-linux-gnueabihf
+		# fpu according to cross-rs
```

Comment:
Is there a link to the documentation or code that means that we need this fpu setting?

Reply:
There is [this ](https://github.com/cross-rs/cross/blob/main/docker/Dockerfile.armv7-unknown-linux-gnueabihf#L50)but I am not really sure it's needed. I will do try commit and see.

Reply:
Looks like it's not needed really.